tests/tensor/test_basic.py

```diff
@@ -4392,6 +4393,15 @@ def test_empty():
     assert res.dtype == "int64"
 
 
+def test_identity_like():
```

Comment:
Shall we give it a more specific name?

```suggestion
def test_identity_like_dtype():
```